src/components/pages/metadata-api/metadata-api-presentation.tsx

```diff
@@ -0,0 +1,150 @@
+import clsx from 'clsx'
+
+import { EditorContact } from '../editor/editor-contact'
+import { Link } from '@/components/content/link'
+
+export const MetadataApiPresentation = () => {
+  return (
+    <>
+      <div
+        className={clsx(
+          'md:left-[calc(-50vw+50%)] md:relative',
+          'md:text-left md:ml-0',
+          '-mt-12 sm:text-center',
+          'text-gray-700'
+        )}
+      >
+        <section className="font-bold text-center mt-32 px-4">
+          <h1
+            className={clsx(
+              'text-5xl font-extrabold',
+              'tracking-tight leading-tight',
+              'mt-3 mx-auto'
+            )}
+          >
+            <span className="serlo-underlined !pr-0 pb-2">Metadata API</span>
+          </h1>
+        </section>
+        <section className="text-left mt-16 mb-16 px-4 mx-auto max-w-5xl">
+          <div className="sm:flex sm:justify-between sm:space-x-4">
+            <div className="sm:flex-1 text-xl leading-cozy sm:max-w-[31rem] sm:flex sm:flex-col pr-2">
+              <h2 className="text-gray-700 text-[1.3rem] font-extrabold tracking-tight">
+                <br />
+                Integrate our Digital Learning Resources into your apps
+              </h2>
+              <p className="mt-4">
+                Our new Metadata API makes it easy to integrate thousands of
+                Serlo articles, courses, videos, and interactive exercises.
+              </p>
+              <p className="mt-4">
+                The API is completely free and it only takes a few lines of code
+                to integrate a vast repository of educational resources into
+                your app or website!
+              </p>
+              <CallToAction />
+            </div>
+            <div className="sm:flex-1 mt-8 -mx-side sm:max-w-[32rem] rounded-lg overflow-hidden px-4">
+              <img
+                src="/_assets/img/metadata/metadata.png"
+                alt="A bird flying over a stack of documents"
+              />
+            </div>
+          </div>
+        </section>
+        <section className="bg-orangeBow bg-100% px-2 mt-0 !pt-16 mb-20">
+          <div className="mt-2 pb-16 sm:flex text-center text-xl max-w-4xl mx-auto px-4">
+            <div className="flex-1 mt-5">
+              <b className="text-brand font-handwritten text-4xl">20,000+</b>
+              <br />
+              Educational resources have been created with the{' '}
```

Comment:
```suggestion
               X educational resources are available over the Metadata API
```

You can find X by importing the current staging data in `database-layer` with `yarn mysql:import-anonymous-data` and count the number of overall entries with `yarn fetch EntitiesMetadataQuery ...` or by executing the underlying SQL without limit

Reply:
The goal of this was to link back to the Serlo Editor.
I could add a third point saying "10.000+ educational resources are available over the Metadata API" but I'd leave this the way it is.

Here are some interesting numbers, some of which we could display too.
```
DE 10922
out of which
  applet 144
  article 2967
  course 598
  text-exercise 4088
  text-exercise-group 2691
  video 434
EN 686
ES 553
```

Reply:
> The goal of this was to link back to the Serlo Editor.

I don't understand this -> Why do we want to link to the Serlo editor here / Why do we want to hint people to the Serlo editor? Anyhow then this link wouldn't be understandable for me.

Reply:
I thought for better SEO and because the same people that are interested in the Metadata API might also be interested in other Serlo offerings such as our editor. Would you like me to remove the link or first discuss it with Simon on Friday?